com.unity.netcode.gameobjects/Runtime/Transports/UTP/UnityTransport.cs

```diff
@@ -758,6 +761,32 @@ private int ExtractRtt(NetworkConnection networkConnection)
             }
         }
 
+        private float ExtractPacketLoss(NetworkConnection networkConnection)
+        {
+            if (m_Driver.GetConnectionState(networkConnection) != NetworkConnection.State.Connected)
+            {
+                return 0f;
+            }
+
+            m_Driver.GetPipelineBuffers(m_ReliableSequencedPipeline,
+                NetworkPipelineStageCollection.GetStageId(typeof(ReliableSequencedPipelineStage)),
+                networkConnection,
+                out _,
+                out _,
+                out var sharedBuffer);
+
+            unsafe
+            {
+                var sharedContext = (ReliableUtility.SharedContext*)sharedBuffer.GetUnsafePtr();
+
+                var packetReceived = (float)sharedContext->stats.PacketsReceived;
+                var packetDropped = (float)sharedContext->stats.PacketsDropped;
+                var packetLoss = packetReceived > 0 ? packetDropped/packetReceived : 0;
```

Comment:
Is this right? Should it instead be `packetDropped / (packetReceived + packetDropped)`?

Reply:
As an example, if you send 100 packets and 50 of them are lost, then the calculation above would indicate 50/50 = 100% packet loss, rather than 50/100 = 50% packet loss. I think the second is the one we want 🤔

Reply:
@Rosme, you marked this resolved, was there a resolution to it?

Reply:
The formula is correct. The packetReceived is the total with those that are dropped. So if received and dropped are equal, that mean we have dropped every packet we should have received.

Reply:
Great, thanks for looking into it :)